src/backend/distributed/planner/multi_physical_planner.c

```diff
@@ -547,7 +547,7 @@ BuildJobQuery(MultiNode *multiNode, List *dependentJobList)
 	List *sortClauseList = NIL;
 	Node *limitCount = NULL;
 	Node *limitOffset = NULL;
-	LimitOption limitOption = LIMIT_OPTION_DEFAULT;
+	LimitOption limitOption = LIMIT_OPTION_COUNT;
```

Comment:
Maybe this should not be initialized by some default anymore.  e.g. 
  `LimitOption limitOption;`

but I am not sure at the moment.

Reply:
I think you're right - if limitCount is NULL, limitOption doesn't matter. If limitCount is set, then limitOption is also initalized properly.